web/template/create_document.gohtml

```diff
@@ -0,0 +1,207 @@
+{{ template "page" . }}
+
+{{ define "title" }}{{ if .Error.Any }}Error: {{ end }}Create Document{{ end }}
+
+{{ define "main" }}
```

Comment:
As design may change, I will split this into smaller templates once the design is confirmed